catalog/spatial/wfs/spatial-wfs-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/catalog/common/AttributeDescriptorComparator.java

```diff
@@ -15,12 +15,16 @@
 
 package org.codice.ddf.spatial.ogc.wfs.catalog.common;
 
+import java.io.Serializable;
 import java.text.Collator;
 import java.util.Comparator;
 
 import ddf.catalog.data.AttributeDescriptor;
 
-public class AttributeDescriptorComparator implements Comparator<AttributeDescriptor> {
+public class AttributeDescriptorComparator implements Comparator<AttributeDescriptor>,
+        Serializable {
+
+    private static final long serialVersionUID = -6030331342388630960L;
```

Comment:
We should use something like `1L` here, not the version assigned by the compiler/IDE.